### PR TITLE
disable grafana integration

### PR DESCRIPTION
**Pull request by a-thaler**
**Description**
To finalize https://github.com/kyma-project/kyma/issues/16306 the static links about observability should be disabled and later removed from the code base.
Also the warning introduced by https://github.com/kyma-project/kyma-dashboard/pull/238 needs to get reverted.

Changes proposed in this pull request:

- disable observability links
- remove grafana deprecation warning
- ...

**Related issue(s)**
https://github.com/kyma-project/kyma-dashboard/pull/238
https://github.com/kyma-project/kyma/issues/16306

### CODE REVIEW

File: environments/dev/config.yaml [hunk 1]
```diff
@@ -52,7 +52,7 @@ config:
     SHOW_GARDENER_METADATA:
       isEnabled: true
     OBSERVABILITY:
-      isEnabled: true
+      isEnabled: false
```
Comment (mrCherry97) [inline environments/dev/config.yaml:55]: if `OBSERVABILITY` is fully removed please remove all this section from config.

File: environments/prod/config.yaml [hunk 2]
```diff
@@ -51,7 +51,7 @@ config:
     SHOW_GARDENER_METADATA:
       isEnabled: true
     OBSERVABILITY:
-      isEnabled: true
+      isEnabled: false
```
Comment (mrCherry97) [inline environments/prod/config.yaml:54]: if `OBSERVABILITY` is fully removed please remove all this section from config.

File: environments/stage/config.yaml [hunk 3]
```diff
@@ -49,7 +49,7 @@ config:
     SHOW_GARDENER_METADATA:
       isEnabled: true
     OBSERVABILITY:
-      isEnabled: true
+      isEnabled: false
```
Comment (mrCherry97) [inline environments/stage/config.yaml:52]: if `OBSERVABILITY` is fully removed please remove all this section from config.

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

File: environments/dev/config.yaml (hunk 1 above)
Comment (a-thaler) [reply environments/dev/config.yaml:55]: That observability is special as it is a real "feature". Removing it fully from the code base was too heavy for me and I like to pass it over to you in a follow-up. So I thought disabling it now will be the safest. I don't know what the default behaviour is when removing the config block only, I can imagine that the default is on.

**Review by mrCherry97 — APPROVED** [review]

**Pull request merged**